### ISSUE TITLE
Panic If `balanceOf` Reverts During Flashloan

### ISSUE THREAD

**Issue by g0rk** [fuzzland/ityfuzz#14]
```
thread 'main' panicked at 'assertion failed: 4 * 8 >= slice.len()', /home/h/开发/rust/.cargo/registry/src/index.crates.io-6f17d22bba15001f/primitive-types-0.12.1/src/lib.rs:38:1
stack backtrace:
   0: rust_begin_unwind
             at /rustc/af06dce64bf87ea9206bdf6cff61c144b9ce8458/library/std/src/panicking.rs:577:5
   1: core::panicking::panic_fmt
             at /rustc/af06dce64bf87ea9206bdf6cff61c144b9ce8458/library/core/src/panicking.rs:67:14
   2: core::panicking::panic
             at /rustc/af06dce64bf87ea9206bdf6cff61c144b9ce8458/library/core/src/panicking.rs:117:5
   3: <primitive_types::U256 as core::convert::From<&[u8]>>::from
   4: <ityfuzz::evm::oracle::IERC20OracleFlashloan as ityfuzz::oracle::Oracle<ityfuzz::evm::vm::EVMState,primitive_types::H160,revm::interpreter::bytecode::Bytecode,bytes::bytes::Bytes,primitive_types::H160,primitive_types::U256,alloc::vec::Vec<u8>,ityfuzz::evm::input::EVMInput,ityfuzz::state::FuzzState<ityfuzz::evm::input::EVMInput,ityfuzz::evm::vm::EVMState,primitive_types::H160,primitive_types::H160,alloc::vec::Vec<u8>>>>::oracle
   5: <ityfuzz::feedback::OracleFeedback<VS,Addr,Code,By,Loc,SlotTy,Out,I,S> as libafl::feedbacks::Feedback<I,S>>::is_interesting
   6: <ityfuzz::fuzzer::ItyFuzzer<VS,Loc,Addr,Out,CS,IS,F,IF,I,OF,S,OT> as libafl::fuzzer::Evaluator<E,EM,I,S>>::evaluate_input_events
   7: <(Head,Tail) as libafl::stages::StagesTuple<E,EM,S,Z>>::perform_all
   8: <(Head,Tail) as libafl::stages::StagesTuple<E,EM,S,Z>>::perform_all
   9: <ityfuzz::fuzzer::ItyFuzzer<VS,Loc,Addr,Out,CS,IS,F,IF,I,OF,S,OT> as libafl::fuzzer::Fuzzer<E,EM,I,S,ST>>::fuzz_loop
  10: ityfuzz::fuzzers::evm_fuzzer::evm_fuzzer
  11: cli::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

```

**Comment by shouc** [comment]
can you share an example contract that has `balanceOf` reverts

**Comment by shouc** [comment]
shall be fixed in #15, can you give it a try
